tests/test_focusAnalysis.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of summit_extras.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+from typing import Iterable
+
+import lsst.utils.tests
```

Comment:
Minor point, but shouldn't this import happen after `lsst.summit.utils`?

Reply:
Is this referring to fact I'm importing the tested code after the `mpl.use('Agg')` line? This is because you have to set the matplotlib backend before any other code which imports matplotlib does (hence the `#noqa: E402` line decorators) or people have plots pop up.

Reply:
No, this is a very minor point. Shouldn't the order of imports be:
```
...
lsst.summit.utils
lsst.utils.tests
```

Reply:
Ohhhh, there's no `isort` over here, and I've made minimal effort anywhere to keep things alphabetical, so that didn't even occur to me! Look elsewhere in the package and you'll see the imports are a free-for-all! I'm considering going pre-commit + black + isort etc at some point though, but that's definitely another ticket 🙂